deploy/registration-service/registration-service.yaml

```diff
@@ -96,10 +96,12 @@ objects:
             - name: registration-service
               image: ${IMAGE}
               ports:
-                - containerPort: 8080
-                - containerPort: 8081
-                - containerPort: 8082
+                - containerPort: 8080 # registration service
+                - containerPort: 8081 # proxy 
+                - containerPort: 8082 # proxy metrics
                   name: metrics
```

Comment:
optional - maybe we could rename this to proxy-metrics

Reply:
yes, indeed, although I did not want to bring too many changes here, in case it's used by Konflux 🤷‍♂️